backend/api/Database/Models/MissionTask.cs

```diff
@@ -203,6 +203,6 @@ public enum MissionTaskType
     {
         Inspection,
         Localization,
-        DriveTo
+        ReturnHome
```

Comment:
Does this remove the ability to have DriveTo tasks, ie. tasks where we tell the robot to go somewhere, but we don't give it a target? I think there are several use-cases for these, and DriveTo is already supported in some Isar implementations. Could we instead add a new task type?

Reply:
I checked this with Afonso, DriveTo was created specifcally for the return to home functionality and it is not used anywhere in the code that is not related to return to home. At the time he wanted to change the name but went on vacation/offshore before he could do the refactor

Reply:
That is odd, since missions from echo use it in other parts of their missions. I think we may need to have a bigger discussion around this.

Reply:
Sorry, it seems I have misunderstood. I see now that this renaming will not affect the naming in the IsarTask